lib/parser/resolve-packages.js

```diff
@@ -0,0 +1,36 @@
+const transform = require('./transform'); //eslint-disable-line
+const {spawn} = require('child_process');
+const path = require('path');
+
+function processPromise(child) {
+	return new Promise(function(resolve, reject) { //eslint-disable-line
+		child.addListener('error', reject);
+		child.addListener('exit', resolve);
+	});
+}
+function spawnChild(pkg) {
+	// spawn('npm', ['install', '--save', pkg], { stdio: 'inherit', customFds: [0, 1, 2] });
```

Comment:
Ok, perhaps then later we can run the block conditionally depending on NODE_ENV or so.

For now, would you add a comment on top of this section to explain why is commented, and how in development mode, we can use it?